spirv_cross/src/msl.rs

```diff
@@ -2,11 +2,51 @@ use {compiler, spirv, ErrorCode};
 use bindings::root::*;
 use std::ptr;
 use std::marker::PhantomData;
+use std::collections::HashMap;
+use std::ffi::CStr;
 
 /// A MSL target.
 #[derive(Debug, Clone)]
 pub struct Target;
 
+pub struct TargetData {
```

Comment:
I don't think it needs to be public? at least the members should be used elsewhere

Reply:
Yeah, I would prefer not to expose this if possible. Without the `pub` here I get `[rustc] private type `msl::TargetData` in public interface`, but it's unclear how to restrict the visibility of an associated type for the `spirv::Target` trait. Is there another way to do this?

Reply:
No, I think `rustc` is right. We should, however, hide the members

Reply:
Agreed, I didn't notice they were left as `pub`. Fixed.